sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/SubscriptionDescription.java

```diff
@@ -175,6 +181,117 @@
             namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
     private EntityAvailabilityStatus entityAvailabilityStatus;
 
+    static {
+        // This is used by classes in different packages to get access to private and package-private methods.
+        EntityHelper.setSubscriptionAccessor(new EntityHelper.SubscriptionAccessor() {
+            @Override
+            public void setTopicName(SubscriptionDescription subscriptionDescription, String topicName) {
+                subscriptionDescription.setTopicName(topicName);
+            }
+
+            @Override
+            public void setSubscriptionName(SubscriptionDescription subscriptionDescription, String subscriptionName) {
+                subscriptionDescription.setSubscriptionName(subscriptionName);
+            }
+        });
+    }
+
+    private String topicName;
+    private String subscriptionName;
+
+    /**
+     * Json deserialization constructor.
+     */
+    @JsonCreator
+    SubscriptionDescription() {
+    }
+
+    /**
+     * Creates an instance with the name of the subscription and its associated topic. Default values for the
+     * subscription are populated. The properties populated with defaults are:
+     *
```

Comment:
Should we put default values as well in documentation , so user know what are the values?.

Reply:
I'll add those.